pom.xml

```diff
@@ -98,6 +127,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>jackson2-api</artifactId>
+            <version>2.12.4-rc199.cb832da5f764</version> <!-- TODO https://github.com/jenkinsci/jackson2-api-plugin/pull/82 -->
```

Comment:
https://github.com/jenkinsci/jackson2-api-plugin/pull/82 suggest marking as draft until that is released

Reply:
I don't think that's necessary. When I was ready to move the PR out of draft status, I clicked on the "Ready to Review" button in GitHub (not the "Ready to Merge" button, which doesn't exist). I know from experience that people are less likely to review a change in draft status. You know that as well, because you did the same for the JEP-228 PR (even though it still had an incremental reference).

Reply:
the change isn't merge-able in it's current state, other labels could signify that, but draft does convey a good signal and prevents merging in the UI.

(as soon as this is updated to not use an incremental and the build is green i'll merge this)

Reply:
> draft does convey a good signal

And also conveys a good signal to reviewers that they shouldn't spend time and effort reviewing a change that might not even have been tested.

I prefer not to send that signal to reviewers. I like fast reviews and I like for my changes to be merged quickly. There's a reason I'm (by far) the most prolific programmer at my company.

Reply:
It is true that it would be nice to have some intermediate status in GH: ready for review but not ready to merge.

Reply:
```suggestion
            <version>2.12.4</version>
```